services/analyzer/sequelize-tables-analyzer.js

```diff
@@ -299,6 +299,26 @@ async function createTableSchema({
   };
 }
 
+function detectAndFixAliasConflicts(wholeSchema) {
```

Comment:
- As _detecting_ is a part of _fixing_, we could consider renaming this function to `fixAliasConflicts` (maybe it could then be more obvious that it has side effects since the first word is `fix`).
- We could have added a notice to explain what specific case we fix here. Reading the code (without digging), I can't see at first glance that the problem is: "when a field as the same name as a relationship, we fix the relationship name". The changelog entry is explicit, maybe we could use something similar (disclaimer: I love comments).
 - (side note) I tried to re-implement this function in a more functional way (10 minutes), but I failed (it was less expressive and understandable), I prefer yours!

Reply:
I updated the name and added a notice 👍